HUD/BeginTextCommandIsThisHelpMessageBeingDisplayed.md

```diff
@@ -9,14 +9,19 @@ aliases: ["0x0A24DA3A41B718F5"]
 void BEGIN_TEXT_COMMAND_IS_THIS_HELP_MESSAGE_BEING_DISPLAYED(char* labelName);
 ```
 
-```
-BOOL IsContextActive(char *ctx)  
-{  
-	BEGIN_TEXT_COMMAND_IS_THIS_HELP_MESSAGE_BEING_DISPLAYED(ctx);  
-	return END_TEXT_COMMAND_IS_THIS_HELP_MESSAGE_BEING_DISPLAYED(0);  
-}  
-```
-
 ## Parameters
-* **labelName**: 
+* **labelName**: Text entry name. This can be created using [#_0x32CA01C3](AddTextEntry).
```

Comment:
This should be the other way around, you can replace it with what I wrote down below if you want.

```
* **labelName**: Text entry name. This can be created using [AddTextEntry](#_0x32CA01C3).
```

Reply:
A new commit with the corrected description is uploded. Thanks for pointing out that mistake!